src/utils/add-opacity.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Adds opacity to the input hex color. Should be used with colors from the
+ * design system, as that is the hex format this function supports. It doesn't
+ * work with 3 letter hex codes, or hex codes already containing the
+ * opacity/alpha channel.
+ *
+ * Example usage:
+ * addOpacity(theme.static.background.background_0.background, 0.2)
+ */
+export const addOpacity = (hex: string, opacity: number) => {
```

Comment:
Should we add a safety check and maybe also a warning if the hex string is not of the required format?

Reply:
Good suggestion! I added a safety check and fallback to return the hex value unopacitiated (that's surely a word). Didn't add Bugsnag warning since that would really spam Bugsnag if this happened.

Reply:
We definitely need to add `unopacitiated` into the vocabulary

Reply:
Nice! I was thinking of just a `console.warn`, but the safety check should be enough :+1: